lib/makeTwemojiRenderer.js

```diff
@@ -60,4 +53,29 @@ function merge() {
 		}
    }
    return out
+}
+
+function getSource(options, icon){
+	var source = ''
+
+	if (options.baseUrl.indexOf('http') !== 0) {
+		source += normalizeProtocol(options.protocol)
+	}
+
+	var baseUrl = options.baseUrl.endsWith('/') ? options.baseUrl : options.baseUrl + '/'
+
+	source += baseUrl
+
+	if(options.size === '' || options.size == null){
+		if(options.ext === '.svg'){
+			//Handle https://github.com/appfigures/react-easy-emoji/issues/25
+			source += 'svg/'
```

Comment:
Instead of injecting this into the path, we should let the user manage it through the `baseUrl` param as the project README shows

Reply:
https://github.com/appfigures/react-easy-emoji/pull/26/commits/dec646eb76faaf4e3d0ba73546f647b4ee3c7856